src/yunohost/log.py

```diff
@@ -475,8 +525,10 @@ def close(self, error=None):
         self.ended_at = datetime.utcnow()
         self._error = error
         self._success = error is None
+
         if self.logger is not None:
             self.logger.removeHandler(self.file_handler)
+            self.file_handler.close()
```

Comment:
Is it like an unrelated fix? Or to prevent from appearing as a parent while things are finished?

Reply:
It's a related fix. Without this, the current code doesnt close the file even after `removeHandler()` and the strategy of looking at `proc.open_files()` doesnt work (well, it works but could catch the wrong file)